### PR TITLE
UHM-6055, each selected drug should have an amount entered

### PR DESCRIPTION
**Pull request by cioan**

### CODE REVIEW

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @cioan !

**Comment by cioan** [comment]
Thanks @mogoodrich !

File: configuration/messageproperties/messages_zl_en.properties [hunk 1]
```diff
@@ -32,6 +32,7 @@ zl.days=Days
 
 zl.dispensingMustBeWithinOneYearOfEncounter=Next dispensing date has to be within a year of the encounter date
 zl.pleaseSelectADrug=Please select a drug
+zl.select.drug.qty=Make sure each selected drug has a corresponding quantity entered
```
Comment (mseaton) [inline configuration/messageproperties/messages_zl_en.properties:35]: @cioan / @mogoodrich - what is the plan for getting corresponding French translations added?  Is there a process for identifying and doing these?

Comment (mogoodrich) [reply configuration/messageproperties/messages_zl_en.properties:35]: Good point @mseaton ... @cioan can we flag @dmdesimone  on this?

Comment (dmdesimone) [reply configuration/messageproperties/messages_zl_en.properties:35]: just added these to the list in https://pihemr.atlassian.net/browse/UHM-6027.

**Pull request merged**